contrib/auto-render/auto-render.js

```diff
@@ -61,9 +61,24 @@ const renderElem = function(elem, optionsCopy) {
             elem.replaceChild(frag, childNode);
         } else if (childNode.nodeType === 1) {
             // Element node
-            const shouldRender = optionsCopy.ignoredTags.indexOf(
+            let shouldRender = optionsCopy.ignoredTags.indexOf(
                 childNode.nodeName.toLowerCase()) === -1;
```

Comment:
IE9 doesn't support `classList`. You can do:
```js
// normalize className to use indexOf for searching
const className = ' ' + childNode.className + ' ';
const shouldRender = optionsCopy.ignoredTags.indexOf(
    childNode.nodeName.toLowerCase()) === -1 &&
    optionsCopy.ignoreClasses.every(
        x => className.indexOf(' ' + x + ' ') === -1)
```

Reply:
Thanks for the hint! Didn't know that your support covers this old browser version, too. Will fix it.

Reply:
@ylemkimon I changed the code. Thanks again for the hint! I also learned that every is supported by IE9, so no need to do everything "manually" with the for-loops.